ibllib/io/extractors/base.py

```diff
@@ -67,16 +72,19 @@ def _write_to_disk(file_path, data):
             else:
                 log.error(f"Don't know how to save {file_path.suffix} files yet")
 
-        if isinstance(self.save_names, str):
+        if self.save_names is None:
```

Comment:
Here I had to distinguish the case were none of the output were requested, all, and partial